src/components/Form.js

```diff
@@ -140,6 +147,12 @@ const reducer = (draft, action) => {
       payload: submission,
     });
 
+    // Load the google script that makes ReCAPTCHA work
+    dispatch({
+      type: 'LOAD_RECAPTCHA',
+      payload: config.reCaptchaSiteKey
+    });
```

Comment:
Please do not use dispatch for that, but write a custom hook which is built using `useEffect` hook. Probably you'll also need `useRef` to keep track of the script element.